packages/flutter_app/ios/ci_scripts/ci_post_clone.sh

```diff
@@ -34,6 +37,6 @@ curl -sL https://firebase.tools | bash
 dart pub global activate flutterfire_cli
 
 # Build the iOS app in release mode with no code signing.
-flutter build ios --release --dart-define-from-file=dart_defines/prod.env --no-codesign
+flutter build ios --release --dart-define-from-file=dart_defines/${flavor}.env --no-codesign
```

Comment:
I apologize for the inconvenience after the review, but I modified the app to allow for building according to the environment by using environment variables!

Reference: https://zenn.dev/link/comments/2eea7c57c6d14b